src/components/app.js

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import logo from '../images/logo.png';
 
-const TestPage = () => (
+const App = () => (
```

Comment:
This component should be given a different name to avoid having two components with the same name which will cause confusion

Reply:
worked on it